### PR TITLE
Edits based off of my blunders

### PR DESCRIPTION
**Pull request by shannonbux**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 5ea74dd51a04a29ddcf2852f59f4f1e527cf0d5f

https://app.netlify.com/sites/using-glamor/deploys/59dfeddb0b79b773f5be498c

**Comment by shannonbux** [comment]
I just corrected the two hyperlinks I had done wrong! It should look good now.

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Let's check first that you have everything setup to start creating with Gatsby.
 
 Node.js is a programming tool for running JavaScript on servers and in your computer's terminal. Gatsby is built using Node.js.
 
-Open a terminal window and type `node --version` then `npm --version`.
+Open a <a href="http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/">terminal window</a> and type `node --version` then `npm --version`.
```
Comment (KyleAMathews) [inline docs/tutorial/part-one/index.md]: This is for macs. We should also include instructions for opening a terminal in windows. Searching around this looks good https://www.quora.com/How-do-I-open-terminal-in-windows

File: docs/tutorial/part-one/index.md [hunk 2]
```diff
@@ -70,9 +70,11 @@ This is huge because it makes development so much faster and fun.
 
 Let's try it.
 
-Try changing "Hello world!" in the page component to "Hello Gatsby!". You'll find the page component at `src/pages/index.js`. The text in your browser should change within a second.
+You'll need code editing software for the next part of this tutorial. <a href="https://code.visualstudio.com/">VS Studio</a>  is a nice one. After downloading and opening a code editing software, open a folder on your computer called "tutorial-part-one." This folder was automatically created when you ran the previous terminal commands in the first part of this tutorial.
 
-Try some other tricks.
+Now that you've opened the "tutorial-part-one" folder in your code editing software, it's time to edit your website. In the code editing software, you'll need to find the 'src/pages/index.js' page. Once you find that, try changing "Hello world!" in the page component to "Hello Gatsby!". The text in your browser should change within a second.
```
Comment (KyleAMathews) [inline docs/tutorial/part-one/index.md]: how about "to find the file at `src/pages/index.js`"

Comment (KyleAMathews) [reply docs/tutorial/part-one/index.md]: In markdown, you can make text appear as inline code by surrounding it with backticks. I'd do that around "src/pages/index.js".

**Comment by KyleAMathews** [comment]
I'll post the link to the preview build for gatsbyjs.org once it's done.

**Comment by KyleAMathews** [comment]
https://deploy-preview-2437--gatsbyjs.netlify.com/tutorial/part-one/

Looks like your html links aren't working for some reason. Which is weird since I thought we supported them… hmmm, anyways, using markdown links is great https://help.github.com/articles/basic-writing-and-formatting-syntax/#links

**Comment by shannonbux** [comment]
ok. I figured out what I did wrong and fixed it. Is it working now?

**Comment by KyleAMathews** [comment]
Looks good! 

This is Github's rendering of it. The "deploy-preview" link I posted in a previous comment will be updated shortly with your changes. 
https://github.com/gatsbyjs/gatsby/blob/48c5a9ffe99325e45371eec1211ab518a65cb53f/docs/tutorial/part-one/index.md

Could you surround src/pages/index.js in backticks?

**Comment by shannonbux** [comment]
put the backticks in. I was confused and put forward ticks before lol. Thanks for your patience. I'm slowly getting to know markup language!

**Comment by KyleAMathews** [comment]
Practice makes perfect :-)

Thanks for the edits!

**Pull request merged**